src/main/java/org/example/User.java

```diff
@@ -0,0 +1,50 @@
+package org.example;
+
+import java.util.Scanner;
+
+import static tictactoe.Main.checkWinner;
+import static tictactoe.Main.printMatrix;
+
+public class User extends Player{
+    @Override
+    public char[][] play(char[][] matrix, char yourChar) {
+        if(checkWinner(matrix).equals("Game not finished")) {
+            Scanner sc = new Scanner(System.in);
+            System.out.println("Enter the coordinates:");
+            int row = -1;
+            int column = -1;
+            if (sc.hasNextInt() && sc.hasNextInt()) {
+                row = sc.nextInt();
+                column = sc.nextInt();
+            } else {
+                System.out.println("You should enter numbers!");
+                play(matrix, yourChar);
+            }
+            if (row > 3 || row < 1 || column > 3 || column < 1) {
```

Comment:
in general, I like you follow the code arrangement pattern. In most cases it looks good, I mean seperating logical code units with blank line. Here we start new if statement and before 1 line up we had another if/else. It would be good to insert a blank line there, so that if/else block are separated for eye.